linera-core/src/unit_tests/client_tests.rs

```diff
@@ -746,12 +675,8 @@ where
 {
     let mut builder = TestBuilder::new(storage_builder, 4, 1).await?;
     // New chains use the admin chain to verify their creation certificate.
-    builder
-        .add_initial_chain(ChainDescription::Root(0), Amount::ZERO)
-        .await?;
-    let sender = builder
-        .add_initial_chain(ChainDescription::Root(1), Amount::from_tokens(4))
-        .await?;
+    builder.add_root_chain(0, Amount::ZERO).await?;
```

Comment:
Nit: making it a bit more explicit that the admin chain is being created

```suggestion
    let _admin = builder.add_root_chain(0, Amount::ZERO).await?;
```